examples/pytorch/nlp/blendcnn/distillation/eager/README.md

```diff
@@ -8,7 +8,7 @@ This document describes the step-by-step instructions for reproducing PyTorch Bl
 ## 1. Installation
 
 ```Shell
-cd examples/pytorch/nlp/blendcnn/distillation/eager
+cd examples/pytorch/nlp/blendcnn/distillation/fx
```

Comment:
Change eager to fx? then, the path need to update too.

Reply:
no need change

Reply:
no need to change